src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import Page from './components/Page';
```

Comment:
의미상으로 구분되는 경우 공백을 넣어서 구분해주시면 좋아요!

```suggestion
import React, { useState } from 'react';

import Page from './components/Page';
```